tests/test_cloud_init_iso.cpp

```diff
@@ -424,6 +424,66 @@ TEST_F(CloudInitIso, updateCloudInitWithNewEmptyExtraInterfaces)
     EXPECT_FALSE(new_iso.contains("network-config"));
 }
 
+TEST_F(CloudInitIso, updateCloneCloudInitSrcFileWithExtraInterfaces)
+{
+    constexpr std::string_view src_meta_data_content = R"(#cloud-config
```

Comment:
If you are talking about `const std::string src_meta_data_content = R"(str)";` and pass that to string reference. Then I would say it is similar string copy-wise.

Reply:
No, here:
```
    original_iso.add_file("meta-data", std::string(src_meta_data_content));
```

Reply:
One of the below thread discussions covered this partially. Using `std::string_view` is more adaptive in the sense that other types of strings can use it without copies and it can used as complied time string.

In our cases here, some instances only have calling code `std::string`. Then it is more or less the same in terms of copying. To construct `std::string` via `std::string_view`, you need to call the constructor explicitly. In other cases like `constexpr char*` or `const std::string`, the construction is done implicitly. I would not call explicit constructor invocation as cluttering the code. 

But anyway, this is really a minor thing here. I do not mind changing to `const std::string` in the `std::string` only uses `std::string_view` cases. But this file contains more instances like this, so the change will affect more occurrences.

Reply:
Why a `char*` and not a `std::string`?

Reply:
`constexpr char*` kind of keeps things open, so `qstring` and `std::string` can construct from that easily. `const std::string` is bound to `std::string`. `const std::string a = "str"`; needs to construct a std::string anyway and make another `std::string `to refer to it.